docs/package.json

```diff
@@ -4,18 +4,18 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "docs:build": "next build",
+    "build:site": "next build",
     "start": "next start",
     "lint": "next lint"
   },
   "dependencies": {
-    "@blocknote/ariakit": "^0.26.0",
-    "@blocknote/core": "^0.26.0",
-    "@blocknote/code-block": "^0.26.0",
-    "@blocknote/mantine": "^0.26.0",
-    "@blocknote/react": "^0.26.0",
-    "@blocknote/shadcn": "^0.26.0",
-    "@blocknote/xl-multi-column": "^0.26.0",
+    "@blocknote/ariakit": "workspace:*",
+    "@blocknote/core": "workspace:*",
+    "@blocknote/code-block": "workspace:*",
+    "@blocknote/mantine": "workspace:*",
+    "@blocknote/react": "workspace:*",
+    "@blocknote/shadcn": "workspace:*",
+    "@blocknote/xl-multi-column": "workspace:*",
```

Comment:
`workspace:*` is PNPM specific to ensure that the docs & playground only ever reference the locally installed workspace version. Otherwise, it can technically be satisfied by anything matching the range. So, this syntax makes it explicit that we always want pnpm to symlink to the local package instead.